src/components/Api.js

```diff
@@ -0,0 +1,164 @@
+class Api {
+  constructor(options) {
+    this._baseURL = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "d4cadbf6-4b07-471d-8c88-393f36774c1d",
```

Comment:
According to the task you need to pass full `headers` and `baseUrl` into the constructor of `Api`. See here https://disk.yandex.ru/i/J93dbJp7eypXKQ . And in the code you need to use only: `headers: this._headers` and `this._baseUrl` (not to duplicate the code)